src/design-pages/passkey_authentication.rst

```diff
@@ -0,0 +1,318 @@
+Passkey - Authentication in a centralized environment
```

Comment:
A general observation. Now that we have most components implemented, it would make sense to rephrase those parts of the design that speak in potential way to state facts. E.g. instead of "will use X" or "would use X", just state the choice, "implementation uses X".